### PR TITLE
docs: components and documentation in two pages

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/20020

### 💡 Background and solution

image

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Split components page from documentation.  |
| 🇨🇳 Chinese | 将开发文档和组件页面分开。 |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/migration-v4.en-US.md](https://github.com/ant-design/ant-design/blob/split-components/docs/react/migration-v4.en-US.md)
[View rendered docs/react/migration-v4.zh-CN.md](https://github.com/ant-design/ant-design/blob/split-components/docs/react/migration-v4.zh-CN.md)

### CODE REVIEW

**Comment by zombieJ** [comment]
诡异……

https://circleci.com/gh/ant-design/ant-design/142424?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by afc163** [comment]
好了

File: site/theme/static/common.less [hunk 1]
```diff
@@ -46,22 +47,56 @@ a {
...
   }
+
+  > div,
+  > div > div {
+    height: 100%;
+  }
 }
 
 .aside-container {
+  height: 100%;
```
Comment (zombieJ) [inline site/theme/static/common.less]: 这个让左边滚动的时候线条丢了：
Kapture 2019-12-20 at 20 01 11

Comment (afc163) [reply site/theme/static/common.less]: 👍，我来改改

Comment (afc163) [reply site/theme/static/common.less]: 18568be7de

Comment (afc163) [reply site/theme/static/common.less]: 18568be7de

**Pull request merged**